core/lerna/yarn.lock

```diff
@@ -0,0 +1,4648 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Absolutely not. Did you even _read_ the [contributing docs](https://github.com/lerna/lerna/blob/master/CONTRIBUTING.md)?